app/models/dhs1171_pdf.rb

```diff
@@ -14,20 +14,25 @@ class Dhs1171Pdf
     "zip",
   ].freeze
 
-  SOURCE_PDF = "DHS_1171.pdf".freeze
+  PDF_DIRECTORY = "lib/pdfs".freeze
+  SOURCE_PDF = "#{PDF_DIRECTORY}/DHS_1171.pdf".freeze
+  COVERSHEET_PDF = "#{PDF_DIRECTORY}/michigan_snap_fax_cover_letter.pdf".freeze
 
-  def initialize(client_data)
+  def initialize(client_data:, output_filename: "test_output.pdf")
```

Comment:
Was thinking about it a few mins ago, and I think it's right for `save` to return `true` but for us to know the file name because of dependency injection. No need for `save` to return the file name. 

This is what you already have here, just wanted to write out my thinking because before I was unsure